include/jemalloc/internal/lockedint.h

```diff
@@ -80,6 +80,64 @@ locked_dec_u64(tsdn_t *tsdn, malloc_mutex_t *mtx, locked_u64_t *p,
 #endif
 }
 
+static inline bool
+locked_advance_u64(tsdn_t *tsdn, malloc_mutex_t *mtx, locked_u64_t *p,
+    const uint64_t x, const uint64_t interval) {
+	LOCKEDINT_MTX_ASSERT_INTERNAL(tsdn, mtx);
+	uint64_t before, after;
+	bool overflow;
+	/*
+	 * If this function is called with a large x, extreme overflow
```

Comment:
This comment doesn't make sense contextually; I think it probably should stay with the counter code.